convert-hf-to-gguf.py

```diff
@@ -1629,6 +1633,94 @@ def write_tensors(self):
                 self.post_write_tensors(tensor_map, name, data_torch)
 
 
+class BertModel(Model):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.block_count = self.hparams["num_hidden_layers"]
+
+    def set_gguf_parameters(self):
+        # TODO(cebtenzzre): merge with parent class
```

Comment:
Note to self: resolve this before merge

Reply:
have you... have you forgotten about this..